c++/src/kj/async-test.c++

```diff
@@ -25,14 +25,7 @@
 #include "mutex.h"
 #include "thread.h"
 
-// TODO(later): Refactor this more cleanly so that the KJ library itself defines this.
-#if !__BIONIC__ && !KJ_NO_EXCEPTIONS
-#define KJ_FIBERS_AVAILABLE 1
-#else
-#define KJ_FIBERS_AVAILABLE 0
-#endif
-
-#if !KJ_FIBERS_AVAILABLE
+#if !KJ_USE_FIBERS
 // For bionic, OpenBSD, and no-exception builds, enables the regression test for
```

Comment:
@vlovich This comment seems to be somewhat stale now. Not sure if I just need to remove it?

Reply:
Yeah, I'd remove this.

Reply:
Removed.